coderd/healthcheck/derphealth/derp.go

```diff
@@ -32,6 +32,8 @@ const (
 	warningNodeUsesWebsocket = `Node uses WebSockets because the "Upgrade: DERP" header may be blocked on the load balancer.`
 	oneNodeUnhealthy         = "Region is operational, but performance might be degraded as one node is unhealthy."
 	missingNodeReport        = "Missing node health report, probably a developer error."
+	noSTUN                   = "No nodes are capable of STUN, direct connections may not be possible."
+	stunMapVaryDest          = "STUN detected variable mapping by destination IP, you may be behind a hard NAT."
```

Comment:
I have no idea what I'm reading... so do users? For example, `direct connections may not be possible` clearly describes what does not work. Should we rephrase it to make it user-friendly?